about.html

```diff
@@ -0,0 +1,147 @@
+                </button>
+                <a class="navbar-brand" href="#">Brand</a>
+              </div>
+
+              <!-- Collect the nav links, forms, and other content for toggling -->
+              <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right">
+
+                  <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">About the Fund <span class="caret"></span></a>
+                    <ul class="dropdown-menu" role="menu">
+                      <li><a href="#">About the Fund</a></li>
+                      <li><a href="#">Board &amp; Staff</a></li>
+                      <li><a href="#">Annual Report</a></li>
+                      <li><a href="#">Staff Resources</a></li>
+                    </ul>
+                  </li>
+                  <li><a href="#">Our Initiatives</a></li>
+                  <li><a href="#">Media</a></li>
+                  <li><a href="#">Get Involved</a></li>
+                </ul>
+              </div><!-- /.navbar-collapse -->
+            </div><!-- /.container-fluid -->
+          </nav>
+        </div>
+  	</header>
+    
+    <main>
+    	<section class="firsthing">
+    		<div class="container">
+    			<div class="page-header">
+    				<h1><small>About the Fund</small></br>About the Fund</h1>
+    			</div>
+    		</div>
+    	</section>
+
+    	<article>
+    		<div class="container">
+    			<div class="row">
+    				<div class="col-sm-8">
+    					<h2>Our Mission</h2>
+    					<p class="lead">The Mission of the Mayor’s Fund for Philadelphia is to advance the Mayor's goals through leveraging public-private partnerships to the benefit of the citizens of Philadelphia.</p>
+    					<h3>Our Values</h3>
+    					<p>The Mayor’s Fund for Philadelphia (Board, Staff and Volunteers) is committed to fully integrating the following values into all that it does. These are the principles that will guide our work on behalf of the Mayor and for the residents of Philadelphia:</p>
+    					<h4>Stewardship</h4>
+    					<p>The Fund is committed to a superior level of stewardship of all public and private funds, with the highest standard of integrity, transparency and efficiency.</p>
+    					<h4>Equality</h4>
+    					<p>We seek to integrate equality in all that we do as an employer, funder, and in our role as a point of entry for public-private partnerships in Philadelphia.</p>
+    					<h4>Collaboration</h4>
+    					<p>The Fund will facilitate collaboration between Philadelphia’s public, private and nonprofit sectors to ensure a successful and prosperous Philadelphia.</p>
+    					<h4>Excellence</h4>
+    					<p>The Fund is committed to excellence in all that it does, and to always go the extra step for the residents of Philadelphia.</p>
+    				</div>
+    				<aside class="col-sm-4 history">
```

Comment:
I would wrap the content inside `col-sm-4` with the `history` class, instead of putting it on the column (easier to style without affecting the column's styles).